workflow/rest/runs/logs.mdx

```diff
@@ -0,0 +1,340 @@
+---
+title: "List events"
+description: "Fetch flat event logs of workflow runs"
+api: "GET https://qstash.upstash.io/v2/workflows/events"
+authMethod: "bearer"
+---
+
+<Warning>
+  The flat events provide a more detailed version of the [List Workflow Events API](/workflow/rest/runs/event-logs). If you need an overview of a workflow run, that API is more useful, as it returns grouped and processed data.
```

Comment:
This link needs to be updated.
`/workflow/rest/runs/event-logs` -> `/workflow/rest/runs/runs`